lib/karafka/callbacks/config.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Karafka
+  module Callbacks
+    # Additional configuration required to store procs that we will execute upon callback trigger
+    module Config
+      # Builds up internal callback accumulators
+      # @param klass [Class] Class that we extend with callback config
+      def self.extended(klass)
+        # option internal [Hash] - optional - internal karafka configuration settings that should
+        #   never be changed by users directly
+        klass.setting :internal do
+          Callbacks::TYPES.each do |callback_type|
+            # option [Array<Proc>] array of blocks that will be executed in a given moment
```

Comment:
an array of blocks that will be executed at a given moment depending on the callback type